src/main/java/easy/mind/com/api/DTO/conversion/UserDtoToUser.java

```diff
@@ -0,0 +1,29 @@
+package easy.mind.com.api.DTO.conversion;
+
+import easy.mind.com.api.DTO.UserDTO;
+import easy.mind.com.api.Entity.User;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
`@NoArgsConstructor(access = AccessLevel.PRIVATE)`
Utility classes should not be initiated.